android/build.gradle

```
@@ -10,7 +11,9 @@ buildscript {
     classpath 'com.android.tools.build:gradle:3.2.1'
     classpath 'com.google.gms:google-services:3.2.1'
     classpath 'de.undercouch:gradle-download-task:2.0.0'
-    classpath 'com.amazonaws:aws-devicefarm-gradle-plugin:1.3' // https://github.com/awslabs/aws-device-farm-gradle-plugin/releases
+    classpath 'com.amazonaws:aws-devicefarm-gradle-plugin:1.3'
```

I guess the comment should stay here 😅
```suggestion
    classpath 'com.amazonaws:aws-devicefarm-gradle-plugin:1.3' // https://github.com/awslabs/aws-device-farm-gradle-plugin/releases
```